### PR TITLE
feat/added-scrit

### PR DESCRIPTION
**Pull request by SzDabrowski**

### CODE REVIEW

**Review by NerdbordMentor — CHANGES_REQUESTED** [review]
## Code Review Report

### Task Goal
The task was to change the border color of the `.box` element over time.

### Requirements
1. The logic should be written in the `script.js` file.
2. The `setInterval` method should be used with 3 seconds as a parameter.
3. The color should change between at least 4 different colors.

### Review

The provided code changes are in the `script.js` file, which meets the first requirement.

The `setInterval` method is used, but not as specified in the requirements. The requirement is to change the color every 3 seconds, but in the provided code, the interval is set to `(i+1)*3000` milliseconds. This means that the first color will change after 3 seconds, the second after 6 seconds, the third after 9 seconds, and so on. This does not meet the second requirement.

The code does change the border color of the `.box` element between 4 different colors - 'red', 'blue', 'green', 'yellow'. This meets the third requirement.

### Conclusion

The code changes do not fully meet the task requirements due to the incorrect use of the `setInterval` method. The color change interval should be fixed to every 3 seconds as specified in the requirements.

**Review by NerdbordMentor — CHANGES_REQUESTED** [review]
## Code Review Report

### Task Goal
The task goal is to change the `border-color` of the `.box` element over time.

### Requirements
1. The logic should be written in the `script.js` file.
2. The `setInterval` method should be used with 3 seconds as a parameter.
3. It should change between at least 4 different colors.

### Code Changes Review

The provided code changes are in the `script.js` file, which aligns with the first requirement. The `setInterval` method is used with 3 seconds as a parameter, which meets the second requirement. The code changes between 4 different colors: 'red', 'blue', 'green', and 'yellow', which satisfies the third requirement.

However, there is a logical issue in the code. The `setInterval` function is used inside a `forEach` loop, which means that all intervals start at the same time. As a result, the border color will always be set to the last color in the array ('yellow') every 3 seconds, because this interval will be the last one to be set and will override previous ones.

To fix this issue, you could use `setInterval` outside of the loop and increment an index variable every 3 seconds to cycle through the colors array.

Here is an example of how you could implement this:

```javascript
console.log('Hello from Nerdbord!')

const box = document.querySelector(".box");
const colors = ['red','blue','green','yellow'];
let i = 0;

setInterval(function (){
    box.style.borderColor = colors[i];
    i = (i + 1) % colors.length;
},3000);
```

This code will change the border color of the `.box` element every 3 seconds, cycling through the colors in the `colors` array.

**Review by NerdbordMentor — APPROVED** [review]
# Code Review Report

## Task Goal
The task goal was to change the border color of the `.box` element over time.

## Requirements
1. The logic should be written in the `script.js` file.
2. The `setInterval` method should be used with 3 seconds as a parameter.
3. It should change between at least 4 different colors.

## Code Changes Review

The provided code changes in `script.js` file meet all the requirements of the task:

1. The logic is written in the `script.js` file.
2. The `setInterval` method is used with 3 seconds (3000 milliseconds) as a parameter.
3. The border color of the `.box` element changes between 4 different colors: red, blue, green, and yellow.

The code uses a `setInterval` function to change the border color of the `.box` element every 3 seconds. It cycles through an array of 4 colors and applies each color to the border in turn.

In conclusion, the code changes cover all the task requirements.
